src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -312,9 +313,10 @@ export class ItemByIdComponent implements OnDestroy, BeforeUnloadComponent, Pend
       filter(([ previous, current ]) => (!!current && (!previous || isTask(previous.item) || isTask(current.item)))),
       map(([ , current ]) => ensureDefined(current).item),
     ).subscribe(item => {
+      const isInitFromMobile = !this.layoutService.fullFrameValue.animated && this.breakpointObserver.isMatched('(max-width: 479px)');
```

Comment:
I see that fullFrame was initially a observable... so instead of adding a function `fullFrameValue`, you should `withLatestFrom(this.layoutService.fullFrame$)`.

Reply:
Fixed